lib/cc/engine/file_list_resolver.rb

```diff
@@ -30,12 +32,12 @@ def absolute_include_paths
       end
 
       def rubocop_file_to_include?(file)
-        if file =~ /\.rb$/
-          true
-        else
-          root, basename = File.split(file)
-          @config_store.for(root).file_to_include?(basename)
-        end
+        root, basename = File.split(file)
+        store = config_store.for(root)
+
+        return false if store.file_to_exclude?(basename)
```

Comment:
I don't love it. With a negation and both `&&` and `||` involved in the expression, I prefer to break it apart.